scss/4-elements/_tables.scss

```diff
@@ -44,6 +49,15 @@
     display: none;
   }
 
+  p {
+    font-size: $cads-font-size;
+    margin: 0;
+  }
+
+  p + p {
+    margin-top: 0.75em;
```

Comment:
Should this be a standard spacing? Normal `p` spacing is 1.5rem (`$cads-blah-5`)

Reply:
probs should look at adding the standard line height too
maybe just mixin `%cads-paragraph` ?